share/spack/gitlab/cloud_pipelines/stacks/e4s-cray-rhel/spack.yaml

```diff
@@ -8,6 +8,10 @@ spack:
     reuse: false
     unify: false
 
+  config:
+    shared_linking:
+      missing_library_policy: ignore  # due to use of externals
```

Comment:
Could we traverse a specs dependencies and add all external dependency prefixes to the ignore list? It would be nice to still have this check everything else.

Reply:
That's almost equivalent to disabling the feature, given that glibc has a prefix /usr.

Reply:
I agree as it is now that is effectively what would happen. Thinking more on this we would have to know in advance the name of all of the libraries to exclude for each external, which we don't necessarily, and then have per-library allow prefixes which is a lot more to set up and track.

I think for a first pass this is fine, we get a lot of the way there to detecting issues with Spack only installs. This feature is not quite as useful for HPC deployments as a lot of the core libs are external there, but the situation is also not worse after this change either.